AIPscan/Reporter/templates/reports.html

```diff
@@ -65,48 +65,68 @@
 
       <tr>
           <th>Report</th>
-          <th>Additional parameters</th>
+          <th>Parameters</th>
           <th>Format</th>
       </tr>
 
       <tr>
-          <td>AIP contents</td>
-          <td></td>
-          <td><a href="#"><button type="button" id="report2a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
+          <td>
+              <strong>AIP storage locations</strong>
+              <br>
+              <span class="text-muted">Number of AIPs and their storage volume, per storage location</span>
+          </td>
+          <td><span class="text-muted">n/a</span></td>
+          <td>
+              <a href="#"><button type="button" id="storageLocations" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a>
+          </td>
       </tr>
 
       <tr>
-          <td>File format count</td>
-          <td>Uses date fields</td>
-          <td><a href=""><button type="button" id="report1a" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a><a href=""><button type="button" id="report1b" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Pie Chart</button></a><a href=""><button type="button" id="report1c" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Scatter Plot</button></a></td>
+          <td><strong>AIP contents</strong>
+          <br>
+          <span class="text-muted">Types of file formats in an AIP, including file count per format</span>
```

Comment:
This reports on more than just the file counts, though that is a substantial part of it. I think this branch needs to be rebased on `main` to include 471a160b3b221a5f1221d3bf81dcb8f325e361ff because I don't see those changes here.

In that commit, I settled on the description "Condensed overview of AIPs, including sorted list of file format versions by PUID" for AIP Contents after trying a few permutations.

Reply:
Trying to keep word count to a minimum and word usage consistent across descriptions, I settled on: "Types of file formats in an AIP, including file count per format."